classes/security/password.inc.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * @see https://github.com/ircmaxell/password_compat
+ */
+
+// password_compat uses namespaces, so ensure PHP version supports this
+if (version_compare(phpversion(), '5.3.0', '>=')) {
+	require_once(BASE_SYS_DIR . '/lib/password_compat/lib/password.php');
+}    
+
+// set a flag to indicate whether modern encryption is supported
+define('LEGACY_ENCRYPTION', !(
```

Comment:
Rather than making this a defined constant, I think it would be cleaner to make it a function in the Validation class.